ui/components/ui/add-network/add-network.component.js

```diff
@@ -0,0 +1,96 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { I18nContext } from '../../../contexts/i18n';
+import ActionableMessage from '../actionable-message';
+
+const AddNetwork = ({
+  onBackClick,
+  onAddNetworkClick,
+  onAddNetworkManuallyClick,
+  featuredRPCS,
+}) => {
+  const t = useContext(I18nContext);
+
+  const onBack = () => {
+    onBackClick();
+  };
+
+  const onAddNetwork = () => {
+    onAddNetworkClick();
+  };
+
+  const onAddNetworkManually = () => {
+    onAddNetworkManuallyClick();
+  };
+
+  const nets = featuredRPCS
+    .sort((a, b) => (a.ticker > b.ticker ? 1 : -1))
+    .slice(0, 5);
+
+  return (
+    <div className="add-network__wrapper">
+      <div className="add-network__header">
+        <img
+          src="./images/caret-left-black.svg"
+          alt=""
+          onClick={onBack}
+          className="add-network__header__back-icon"
+        />
+        <div className="add-network__header__title">{t('addNetwork')}</div>
+      </div>
+      <div className="add-network__content">
+        <div className="add-network__content__text">
+          {t('addFromAListOfPopularNetworks')}
+        </div>
+        <div className="add-network__content__networks-title ">
```

Comment:
[REQ] first, we need to use semnatic HTML elements for containing text, not divs. If this is a title it likely should be an h2 or some level of h tag. Secondly, the `<Typography>` component can completely replace the need for using custom styles. Please use it for all text styling.

Reply:
Fixed in: bf446bf677fefe3b46b4fb3b60630f531939dd36